static/js/containers/FinancialAidCalculator.js

```diff
@@ -109,6 +124,16 @@ const validationMessage = (key, errors) => {
   </div>;
 };
 
+const apiError = ({ message, code }: FetchError) => (
+  <div className="api-error">
```

Comment:
We're not actually using a `<form>` tag here either, not sure if that changes things.